core/templates/dev/head/css/oppia.css

```diff
@@ -3989,6 +3989,16 @@ md-card.preview-conversation-skin-supplemental-card {
   margin-right: 2px;
   vertical-align: initial;
 }
+/* Accessibility workaround for icon buttons and links
+  Taken from:
+  https://www.nczonline.net/blog/2013/04/01/making-accessible-icon-buttons/ */
+.oppia-icon-btn-label {
```

Comment:
The scope will be for all icon buttons and links. How about .oppia-icon-accessibility-label? Or is that still confusing?